_posts/fr/2022-09-29-mission-accessible.md

```diff
@@ -0,0 +1,206 @@
+ 
+### - Niveau AAA, le véritable engagement
+ 
+Le niveau AAA, avec 28 critères de plus que le niveau précédent, est très restrictif et n'est atteint que sur des sites hautement spécialisés. Du point de vue de la perception (_Perceivable_), **il permet une très petite gamme de nuances de couleurs avec un contraste très élevé**, les options de conception sont donc assez limitées.
+ 
+Une dynamique parfaitement compréhensible et flexible est nécessaire pour se conformer à ce niveau. Le point unique en (O) indique que l'application peut être entièrement naviguée au clavier sans exception, mais il existe de nombreuses spécifications autour de la section Compréhensible (_Understandable_) pour gérer, par exemple, la soumission de données dans un formulaire : il doit être modifiable , appliquer une vérification lors de l'achèvement de la saisie afin de fournir la possibilité de corriger les erreurs et également d'afficher une confirmation avant la soumission.
+ 
+### Vers quel niveau viser ?
+ 
+Tout d'abord, le fait que tous les critères doivent être réunis pour atteindre un niveau **ne doit décourager aucun produit web**. Les efforts doivent être mesurés en fonction du public cible et du type de site Web. Parfois, il y aura des lois qui peuvent pénaliser l'institution ou l'entreprise si elles ne sont pas respectées. Parfois, les frontières sont moins définies et la rigueur est plus lâche.
+ 
+D'autre part, de nombreux critères d'accessibilité ne font que suivre les bonnes pratiques. Cela rend notre code plus propre et plus performant. Cela rend notre conception plus claire et améliore notre UX. Ensuite, et probablement le plus important de tout, **un Web plus accessible rend l'Internet plus démocratique et plus juste**. Alors pourquoi minimiser l'effort ?
+ 
+## Comment évaluer son produit ?
+ 
+Une fois les principes appliqués se pose la question de l’évaluation. Est-ce que votre produit est effectivement davantage accessible pour les contextes d’usages que vous envisagiez ?
+ 
+<div style="text-align: center; margin: 2rem 0;">
+   <img src="{{ site.baseurl }}/assets/2022-09-29-mission-accessible/guidelines.png" width="300px" alt="Illustration of a guidelines book" style="display: block; margin: auto;"/>
+</div>
+ 
+### Outils d’audit
+ 
+Il existe de nombreux outils qui peuvent être utiles pour évaluer le niveau d’accessibilité de votre produit. Parmi eux, nous pouvons citer les outils suivant qui permettent d’évaluer:
+ 
+-   L’accessibilité globale de votre produit et d’identifier les éventuels problèmes d’accessibilité avec [CodeSniffer](https://github.com/squizlabs/HTML_CodeSniffer), [LightHouse](https://developer.chrome.com/docs/lighthouse/overview/) ou [Accessibility Insights](https://accessibilityinsights.io/);
+-   L’utilisation de vos tags HTML en retirant le CSS de votre page avec [Naked Styles](https://gist.github.com/estudiobold/4181e56129ed5cbd0b6cf40c73787a56);
```

Comment:
tu as parlé de balise dans l'article. par homogénéité, utilise le même mot

```suggestion
-   L’utilisation de vos balises HTML en retirant le CSS de votre page avec [Naked Styles](https://gist.github.com/estudiobold/4181e56129ed5cbd0b6cf40c73787a56);
```

Reply:
done